utrechtteam_vegu2021.html

```diff
@@ -0,0 +1,265 @@
+        </button>
+        <div class="collapse navbar-collapse" id="navbarResponsive">
+            <ul class="navbar-nav ml-auto">
+                <li class="nav-item">
+                    <a class="nav-link" href="#positions">Open Positions</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="#team">Team</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="#socialmedia">Social Media</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="#projects">Projects</a>
+                </li>
+            </ul>
+        </div>
+    </div>
+</nav>
+
+<!-- Page Content -->
+<div class="container">
+
+    <hr>
+
+    <h1 class="display-4" style="text-align: center;">Ocean Parcels team at vEGU2021<a href="https://www.egu21.eu/">
+        <img alt="vEGU2021" src="images/EGU_GeneralAssembly_logo.png" width="300" height="150">
+     </a></h1>
+    
+    <hr>
+
+    <h2 id="team">Presenters</h2>
+    <br>
+    <div class="row justify-content-md-center">
+        <div class="col-lg-3 col-md-4 col-sm-6 mb-3">
+            <div class="card h-100 text-center">
+                <img class="card-img-top team-photo" src="images/SteffieYpma.jpg" alt="">
+                <div class="card-body">
+                    <h4 class="card-title">Stefanie Ypma</h4>
+                    <p class="card-subtitle mb-2 text-muted">Mikael Kaandorp, Jen Jones, Andy Donnelly and Erik van Sebille</p>
+                    <br>
+                    <h5 class="card-subtitle mb-2 text">Mon, 26 Apr, 11:05–11:07</h5>
+                    <p class="card-text">Using machine learning techniques to predict beaching of marine debris on the Galapagos Islands
+                    </p>
+                </div>
+                <div class="card-footer" style="height: 60px">
+                    <a class="card-link" href="https://galapagosplasticfree.nl/egu21/">
+                        <i class="fa fa-info-circle fa-2x"></i>
+                    </a>
+                    <a class="card-link" href="https://meetingorganizer.copernicus.org/EGU21/EGU21-274.html">
+                        <i class="fa fa-external-link fa-2x"></i>
+                    </a>
+                </div>
+            </div>
+        </div>
+
+        <div class="col-lg-3 col-md-4 col-sm-6 mb-3">
+            <div class="card h-100 text-center">
+                <img class="card-img-top team-photo" src="images/DelphineLobelle.jpg" alt="">
+                <div class="card-body">
+                    <h4 class="card-title">Delphine Lobelle</h4>
+                    <p class="card-subtitle mb-2 text-muted">Merel Kooi, Albert A. Koelmans, Charlotte Laufkotter, Cleo E. Jongedijk, Christian Kehl, and Erik van Sebille</p>
+                    <br>
+                    <h5 class="card-subtitle mb-2 text">Mon, 26 Apr, 11:07–11:09</h5>
```

Comment:
Would it work better to put these dates in the footer, so that they are all aligned?